Source/Tests/TestHarnessRunner.cpp

```diff
@@ -62,6 +62,10 @@ void MsgHandler(LogMan::DebugLevels Level, char const *Message) {
 
 void AssertHandler(char const *Message) {
   fmt::print("[ASSERT] {}\n", Message);
+
+  // make sure buffers are flushed
+  fflush(stderr);
```

Comment:
Shouldn't you just use `fflush(nullptr);` to get all streams flushed?

Reply:
done